11 SpinnerAsync/src/components/app.tsx

```diff
@@ -1,10 +1,10 @@
 import * as React from 'react';
 import { createStore, applyMiddleware } from 'redux';
 import { Provider } from 'react-redux';
+import ReduxThunk from 'redux-thunk';
```

Comment:
I think  reduxThunk is better because is the external name and it is more explicit